Template/milestone/table.php

```diff
@@ -1,8 +1,13 @@
 <?php if (! empty($milestone)): ?>
+<div class="">
+
+</div>
 <table class="task-links-table table-stripped">
     <thead>
     <tr>
-        <th class="column-45" colspan="2"><?= t('Title') ?></th>
+        <th><?= t('Project') ?></th>
```

Comment:
The idea of the Milestone view was to kept the "spirit" of the subtask view. This was possible only by keeping it simple, and that's why I finally removed project name and column (and I never tried to add swimlane). But I agree that these are missing. Do you have a screenshot of the result please? (showing also subtasks)

Another idea could be to forget the subtask view and use the list view instead (which contains everything, so great!). 

Any thoughts?

Reply:
That's the result:

![Example](https://user-images.githubusercontent.com/14114209/63372437-7d69bd00-c35c-11e9-8d62-20ebeef4f80b.PNG)

Reply:
I this plugin in my company (a brazilian public agency), for the sprint management and is enough for our needs, but for many projects at the same time showing the project name and column would help. 

Thinking about improvements, would be really useful have a way to order tasks by table columns (datatable like).